callstack/react-native-paper
# Feat: use Material Design Community Icons

This PR introduces the replacement of Material Icons to Material Community Icons. 
Some of the icons have different names, that's why there are so many changes.

### Motivation
#926 
### Test plan

Unfortunately, it needs to be tested visually. 
1.  Open example app and go through all examples to verify if there is no `?` in place of an icon
2. Open all snacks from the documentation page.

I changed all the icons one-by-one so there shouldn't be anything missing, but it can be double checked.
